app/services/katello/pulp3/erratum.rb

```diff
@@ -55,6 +55,14 @@ def update_model(model)
         update_packages(model, backend_data['pkglist']) unless backend_data['pkglist'].blank?
         update_modules(model, backend_data['pkglist']) unless backend_data['pkglist'].blank?
 
+        if !updated && repository.present?
```

Comment:
I have yet to test this, but what would you think about integrating this into the checks above on line 45 & 46?

```ruby
if model.updated.blank ||
  model.attributes.excluding(model.attributes.keys - custom_json.keys) != custom_json
...
```
I don't see any reason why it would have to run after the body of that `if` statement.  I'm thinking it could be the third or check.

Reply:
In my opinion, having the long third 'or' check with parenthesis and split across lines is quite hard to read. That is why I split into 2 'if' which looks cleaner and easier to read and understand in my opinion.

Reply:
Alrighty, I don't feel strongly one way or the other.